scipy/stats/morestats.py

```diff
@@ -1553,6 +1553,18 @@ def bartlett(*args):
     pvalue : float
         The p-value of the test.
 
+    See Also
+    --------
+    fligner : A non-parametric test for the equality of k variances
+    levene : A parametric test for equality of k variances
```

Comment:
I would mention "robust" for levene, as contrast to bartlett
`levene : A robust parametric test for equality of k variances`

what exactly robust means can be read in the docstrings